cmd/system-probe/subcommands/run/command.go

```diff
@@ -240,6 +241,16 @@ func startSystemProbe(cliParams *cliParams, log log.Component, telemetry telemet
 		}()
 	}
 
+	// Setup healthcheck port
+	healthPort := sysprobeconfig.GetInt("system_probe_config.health_port")
```

Comment:
Can you add a `HealthPort int` to the config object in `cmd/system-probe/config/config.go` and populate that in `load()` function of the same file? Then use `cfg.HealthPort` here instead.